### PR TITLE
eyewitness: moving firefox dependency to optdepends

### PR DESCRIPTION
**Pull request by D3vil0p3r**
For users using browsers different from firefox (i.e., chromium, brave, firefox-esr), I would avoid forcing the install of firefox as dependency.

### CODE REVIEW

File: packages/eyewitness/PKGBUILD [hunk 1]
```diff
@@ -12,9 +12,10 @@ license=('GPL3')
 depends=('python' 'python-pyqt4' 'python-netaddr' 'ghost-py' 'python-rdpy'
          'python-selenium' 'python-rsa' 'xorg-server-xvfb'
          'python-beautifulsoup4' 'python-pyvirtualdisplay'
-         'python-fuzzywuzzy' 'python-pillow' 'geckodriver' 'firefox'
+         'python-fuzzywuzzy' 'python-pillow' 'geckodriver'
          'python-easyprocess' 'python-pytesseract')
 makedepends=('git')
+optdepends=('firefox: Selenium module support')
```
Comment (noraj) [inline packages/eyewitness/PKGBUILD:18]: Seems like only firefox is expected

https://github.com/FortyNorthSecurity/EyeWitness/blob/4d7736e18c019a3d3bd43d4f44f212fcae532a3b/Python/setup/setup.sh#L8-L9

https://github.com/FortyNorthSecurity/EyeWitness/blob/4d7736e18c019a3d3bd43d4f44f212fcae532a3b/Python/setup/setup.sh#L66

Comment (D3vil0p3r) [reply packages/eyewitness/PKGBUILD:18]: Currently Yes, but checking the code it seems firefox executable is used mainly for Python Selenium module (I used `grep -rnw ./ -e "firefox"`) . Furthermore, the code tries to install firefox-esr that has a different executable file than firefox, so I think it is a small error of the developer.

Comment (noraj) [reply packages/eyewitness/PKGBUILD:18]: But selenium drives the headless browser required by eyewitness to work right. Can you try to build and run it without firefox on a clean environment with ba-dev to be sure the move is safe?

Comment (D3vil0p3r) [reply packages/eyewitness/PKGBUILD:18]: No firefox binary:
```
#                                  EyeWitness                                  #
################################################################################
#           FortyNorth Security - https://www.fortynorthsecurity.com           #
################################################################################

Starting Web Requests (1 Hosts)
Message: Expected browser binary location, but unable to find binary in default location, no 'moz:firefoxOptions.binary' capability provided, and no binary flag set on the command line

Finished in 0.2802412509918213 seconds
[*] No report files found to open, perhaps no hosts were successful
```
If I use another browser, for example `brave` and I rename its binary as `firefox`, the output is:
```
################################################################################
#                                  EyeWitness                                  #
################################################################################
#           FortyNorth Security - https://www.fortynorthsecurity.com           #
################################################################################

Starting Web Requests (1 Hosts)
Message: Process unexpectedly closed with status 0

Finished in 1.4661085605621338 seconds
[*] No report files found to open, perhaps no hosts were successful
```
If `firefox` is installed, it works correctly.

**Pull request closed without merging**